realm/realm-library/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -118,10 +118,19 @@ public boolean shouldDeleteRealmIfMigrationNeeded() {
         return durability;
     }
 
-    public RealmProxyMediator getSchemaMediator() {
+    RealmProxyMediator getSchemaMediator() {
```

Comment:
fixed.
I added a javadoc comment for this method. please review that. @cmelchior